securitas/templates/user-edit.html

```diff
@@ -1,93 +1,51 @@
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.mail, class="validate") }}
-                </div>
-
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.locale) }}
-                </div>
-              </div>
-
-              <div class="row">
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.ircnick) }}
-                </div>
-
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.timezone) }}
-                </div>
-              </div>
-
-              <div class="row">
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.github) }}
-                </div>
-
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.gitlab) }}
-                </div>
-              </div>
-
-              <div class="row">
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.rhbz_mail) }}
-                </div>
-              </div>
-
-              <!-- Keep this one last for rendering purposes. -->
-              {# TODO: We need to have a way to add more keys #}
-              <div class="row">
-                <div class="input-field col s6">
-                  {{ macros.with_errors(form.gpgkeys) }}
-                </div>
-              </div>
-
-              <div class="row">
-                <div class="input-field col s12">
-                  <button class="btn waves-effect waves-light light-blue darken-4 w100" id="submit" type="submit" tabindex="9">Submit</button>
-                  {{ macros.non_field_errors(form) }}
-                </div>
-              </div>
+          <div class="form-row">
+            <div class="form-group col">{{ macros.with_errors(form.firstname) }}</div>
+            <div class="form-group col">{{ macros.with_errors(form.lastname) }}</div>
+          </div>
+          <div class="form-row">
+            <div class="form-group col">{{ macros.with_errors(form.mail) }}</div>
+            <div class="form-group col">{{ macros.with_errors(form.locale, class="custom-select") }}</div>
+          </div>
+          <div class="form-row">
+            <div class="form-group col">{{ macros.with_errors(form.ircnick) }}</div>
+            <div class="form-group col">{{ macros.with_errors(form.timezone, class="custom-select") }}</div>
+          </div>
+          <div class="form-row">
+            <div class="form-group col">{{ macros.with_errors(form.github) }}</div>
+            <div class="form-group col">{{ macros.with_errors(form.gitlab) }}</div>
+          </div>
+          <div class="form-row">
+              <div class="form-group col">{{ macros.with_errors(form.rhbz_mail) }}</div>
+          </div>
+          <div class="form-row">
+              {# TODO: We need to have a way to add more keys and fix this styling #}
```

Comment:
This part is still broken but it can wait for another PR

Reply:
:thumbsup: